app/Http/Controllers/User/ProfileController.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Http\Controllers\User;
+
+use Exception;
+use App\Models\User;
+use Illuminate\Http\Request;
+use App\Traits\FileProcesser;
+use App\Http\Controllers\Controller;
+use App\Repositories\User\UserInterface;
+use App\Http\Requests\UpdateUserRequest;
+use App\Http\Requests\UploadImageRequest;
+
+class ProfileController extends Controller
+{
+    use FileProcesser;
+
+    /** 
+     * @var userRepository
+     * 
+    */
+    protected $userRepository;
+
+    /**
+     * Create a new controller instance.
+     *
+     * @param  UserInterface  $userRepository
+     * @return void
+     */
+    public function __construct(UserInterface $userRepository)
+    {
+        $this->userRepository = $userRepository;
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     *
+     * @return Illuminate\Http\JsonResponse
+     */
+    public function show($id)
+    {
+        $user = $this->userRepository->find($id);
+
+        return response()->json($user);
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  App\Http\Requests\UpdateUserRequest  $request
```

Comment:
nếu ở trên đã `use App\Http\Requests\UpdateUserRequest;` thì chỗ này chỉ cần viết như sau thôi ạ
```suggestion
     * @param  UpdateUserRequest  $request
```

Reply:
ok em, để anh fix lại :+1: